internal/pkg/flink/internal/controller/input_controller.go

```diff
@@ -128,6 +128,10 @@ func (c *InputController) RunInteractiveInput() {
 			c.isSessionValid(err)
 			continue
```

Comment:
Do we have a test case that gets into this 'if' branch and prints the StatusDetail?

Reply:
Unfortunately no. Our test setup for the input controller is not great at the moment. We haven't found a good way to test the whole STDIN/STDOUT interaction flow, but we have some tickets on our board to address this (current plan is to address it next sprint)

Reply:
Sanity check: in this case this PR would report the detail, right?